src/message_templates/deploy_aws_success.json

```diff
@@ -0,0 +1,63 @@
+{
```

Comment:
If we are going to use the same information as mine, then lets rebase off of my branch